data_ingestion/common/pubsub_publisher.py

```diff
@@ -1,20 +1,7 @@
 import logging
+import os
 from google.cloud import pubsub_v1
 
-
-def notify_data_ingested(project_id, topic, id, gcs_bucket, filename):
-  """Publishes a notification on the notify-data-ingested topic indicating that
-     some data was ingested.
-
-     project_id: The id of the project
-     topic: The name of the topic to notify on
-     id: The id of the source that was ingested
-     gcs_bucket: The name of the bucket the data is located in
-     filename: The name of the file that was uploaded"""
```

Comment:
Since you're removing this function, could you add to the notify_topic documentation that id, gcs_bucket, and filename/fileprefix are expected?